Mathlib/Topology/UniformSpace/UniformEmbedding.lean

```diff
@@ -265,6 +265,21 @@ theorem isComplete_image_iff {m : α → β} {s : Set α} (hm : UniformInducing
   simp_rw [IsComplete, imp.swap (a := Cauchy _), ← mem_Iic (b := 𝓟 _), fact1.forall fact2,
     hm.cauchy_map_iff, exists_mem_image, map_le_iff_le_comap, hm.inducing.nhds_eq_comap]
 
+/-- If `f : X → Y` is an `Inducing` map, the image `f '' s` of a set `s` is complete
+  if and only if `s` is complete. -/
+theorem UniformInducing.isComplete_iff {f : α → β} {s : Set α} (hf : UniformInducing f) :
+    IsComplete s ↔ IsComplete (f '' s) := (isComplete_image_iff hf).symm
```

Comment:
I'm a bit worried that the LHS looks "simpler" than the RHS. Should we just add an alias for `isComplete_image_iff` (or move it to the `UniformInducing` namespace)?

Reply:
I fixed up the two "simpler LHS" lemmas, leaving the `Subtype` one the way it is. Should I also flip the `Inducting.isCompact_iff` parallel lemmas?

Reply:
I'm not too worried about it because they aren't `@[simp]` lemmas